src/Engine/McBopomofoLM.cpp

```diff
@@ -174,22 +175,31 @@ bool McBopomofoLM::externalConverterEnabled() const
 
 void McBopomofoLM::setExternalConverter(std::function<std::string(const std::string&)> externalConverter)
 {
-    m_externalConverter = externalConverter;
+    m_externalConverter = std::move(externalConverter);
 }
 
 void McBopomofoLM::setMacroConverter(std::function<std::string(const std::string&)> macroConverter)
 {
-    m_macroConverter = macroConverter;
+    m_macroConverter = std::move(macroConverter);
 }
 
+std::string McBopomofoLM::convertMacro(std::string input)
```

Comment:
`const std::string& input`.

Reply:
Fixed in 92e46601d055fb57699128d5d2cb8634a1fef339